annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -829,6 +855,11 @@ public void validateWebAnnotation(Annotation webAnnotation) throws ParamValidati
 		case TAGGING:
 			validateTag(webAnnotation);
 			break;
+		case TRANSCRIBING:
+			// if rights are provided, check if it belongs to the valid license list
+			if (webAnnotation != null && webAnnotation.getBody() != null && webAnnotation.getBody().getRights() != null)
```

Comment:
The if statement ca nbe included in the validate method

Reply:
also the name of the right field was changed to edmRights see https://europeana.atlassian.net/browse/EA-1873

Reply:
if statement moved to validate method

Reply:
rights Field changed to edmRights